src/node_contextify.cc

```diff
@@ -211,7 +211,8 @@ MaybeLocal<Context> ContextifyContext::CreateV8Context(
     return MaybeLocal<Context>();
   }
 
-  ctx->SetSecurityToken(env->context()->GetSecurityToken());
+  auto context = env->context();
```

Comment:
```suggestion
  Local<Context> context = env->context();
```

Avoid `auto` for simple types.

Reply:
done.